src/math.jl

```diff
@@ -187,6 +187,14 @@ for (type, base_type, _) in ABSTRACT_QUANTITY_TYPES, f in (:atan, :atand)
         end
     end
 end
+
+# Explicit declaration of `binomial` function.
+function Base.binomial(q1::UnionAbstractQuantity, q2::UnionAbstractQuantity)
```

Comment:
Presumably you would also want to allow for `binomial(Quantity{Int}(3u"1"), 1)`?

Reply:
Right, sorry, still trying to get the hang of this. Does this makes sense?
```julia
# Explicit declaration of `binomial` function.
function Base.binomial(q1::UnionAbstractQuantity, q2::UnionAbstractQuantity)
    iszero(dimension(q1)) || throw(DimensionError(q1))
    iszero(dimension(q2)) || throw(DimensionError(q2))
    return binomial(ustrip(q1), ustrip(q2))
end
function Base.binomial(q1::Any, q2::UnionAbstractQuantity)
    iszero(dimension(q2)) || throw(DimensionError(q2))
    return binomial(q1, ustrip(q2))
end
function Base.binomial(q1::UnionAbstractQuantity, q2::Any)
    iszero(dimension(q1)) || throw(DimensionError(q1))
    return binomial(ustrip(q1), q2)
end
```

Reply:
Instead of `::Any` it should be `::Integer`. Also instead of `UnionAbstractQuantity` it should be `UnionAbstractQuantity{<:Integer}`. Sorry for the confusion.

Reply:
That makes sense, thanks a lot. I guess I should make a similar definition for `factorial` with `UnionAbstractQuantity{<:Integer}` input?

Reply:
Good point, yeah it might make sense to split out.